src/Core/cldj_osa_sub_mod.F90

```diff
@@ -9,18 +9,23 @@ MODULE OSA_SUB_MOD
 
    ! >>>corrected parameter XWRC for < 400 nm from 0 to 0.2
 
-      IMPLICIT NONE
-      PUBLIC  :: OSA
-      CONTAINS
+   IMPLICIT NONE
+
+   PUBLIC :: OSA
+   
+   CONTAINS
 
 SUBROUTINE OSA(WAVEL,WIND,CHLORa, Cangles, OSA_dir)
 
 IMPLICIT NONE
-! number of wavelength (200nm - 4000nm by 10nm apart)
+
+! Number of wavelength (200nm - 4000nm by 10nm apart)
 INTEGER, PARAMETER   :: NNWL = 381
-! absorption coefficients for seawater et 440 nm
+
+! Absorption coefficients for seawater et 440 nm

```

Comment:
@lizziel @pratherUCI: Am wondering if these parameters should be `REAL*8` and if numerical constants should use `d` exponents.  Otherwise we would potentially incur numerical roundoff.  I would leave it as-is for now but maybe we can investigate more in the future.